docs/topics/clusterdefinitions.md

```diff
@@ -52,7 +52,8 @@ $ aks-engine get-versions
-| containerdVersion                 | no (for development only) | Enables an explicit containerd version, e.g. `1.1.4`. Default is `1.1.5`. This `kubernetesConfig` property is for development only, and applies only to cluster creation: `aks-engine upgrade` will always statically set `containerdVersion` to the default version at the time of upgrade, to ensure that upgraded clusters have the most recent, validated version of containerd. This value is currently ignored for Windows.                                                                                                                                                                                                                                                                                                                                                                    |
+| containerdVersion                 | no (for development only) | Enables an explicit containerd version, e.g. `1.1.4`. Default is `1.1.5`. This `kubernetesConfig` property is for development only, and applies only to cluster creation: `aks-engine upgrade` will always statically set `containerdVersion` to the default version at the time of upgrade, to ensure that upgraded clusters have the most recent, validated version of containerd. This value is currently ignored for Windows. It is also ignored when `customContainerdURL` is defined.                                                                                                                                                                                                                                                                                                           |
+| customContainerdURL               | no (for development only) | Allow the use of custom containerd package for Linux nodes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
I assume this is the Linux version of `windowsContainerdURL` (cc @marosset)? If so, let's call this `linuxContainerdURL` to be consistent.

Reply:
Yes, that's the Linux version of `windowsContainerdURL`, will make the change.